### PR TITLE
Fix: Input textfield decimals

### PR DESCRIPTION
**Pull request by EmanueleCitarella**
Task description:
"FUN: F(EXD;*SCO;) 1(;;) 2(MB;SCP_SCH;WETEST_I04) 4(;;SH1).         
Il campo "InputNumber 3 dec" ha 3 decimali (nella colonna del json di as400proxy la lunghezza è impostata correttamente). Il comportamento del campo numerico ketchup fa si che non si riescano a digitare i decimali."

### CODE REVIEW

File: packages/ketchup/src/f-components/f-cell/f-cell.tsx [hunk 1]
```diff
@@ -873,6 +873,7 @@ function setEditableCell(
                     ></input>
                 );
             } else {
+                // console.log(props);
```
Comment (Leonardo-Signorelli) [inline packages/ketchup/src/f-components/f-cell/f-cell.tsx:876]: remove console.log

**Review by Leonardo-Signorelli — CHANGES_REQUESTED** [review]

File: packages/ketchup/src/f-components/f-text-field/f-text-field.tsx [hunk 2]
```diff
@@ -252,14 +260,25 @@ function setContent(props: FTextFieldProps): HTMLDivElement {
...
+                                        valueFromTarget.value,
                                         options,
-                                        true
+                                        true,
+                                        navigator.language
                                     );
+
+                                    valueFromTarget.value = formattedValue;
+                                    props.value = formattedValue;
+                                }
```
Comment (Leonardo-Signorelli) [inline packages/ketchup/src/f-components/f-text-field/f-text-field.tsx:281]: Why did you change valueFromTarget and props.value?

File: packages/ketchup/src/f-components/f-text-field/f-text-field.tsx [hunk 3]
```diff
@@ -452,24 +479,56 @@ function setHelper(props: FTextFieldProps): HTMLDivElement {
     }
 }
 
-const formatValue = function (
+function getMaximumNumbersLength(
```
Comment (Leonardo-Signorelli) [inline packages/ketchup/src/f-components/f-text-field/f-text-field.tsx:482]: These functions are similar to those of the dates. Move the logic into the kup- math. By the way, the kup-math can already support these methods. Please check it out.

**Review by Leonardo-Signorelli — CHANGES_REQUESTED** [review]

File: packages/ketchup/src/f-components/f-text-field/f-text-field.tsx (hunk 2 above)
Comment (EmanueleCitarella) [reply packages/ketchup/src/f-components/f-text-field/f-text-field.tsx:281]: right, I've removed this logic. I've tested and it works

File: packages/ketchup/src/f-components/f-text-field/f-text-field.tsx (hunk 3 above)
Comment (EmanueleCitarella) [reply packages/ketchup/src/f-components/f-text-field/f-text-field.tsx:482]: moved functions in kup-math

**Review by Leonardo-Signorelli — APPROVED** [review]
